hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleNetworkContext.java

```diff
@@ -325,22 +318,22 @@ private void resetUnconditionally(
 		}
 	}
 
-	private String reprOf(RichInstant consensusTime) {
+	private String reprOf(Instant consensusTime) {
 		return Optional.ofNullable(consensusTime)
-				.map(RichInstant::toJava)
+				//.map(RichInstant::toJava)
```

Comment:
Since you don't need to call `RichInstant::toJava` any more, an if then else compare with a `null` should be faster in performance and easier to read.

Reply:
Yeah, changed.